cmd/bench.go

```diff
@@ -15,6 +14,7 @@ import (
 	"testing"
 
 	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
```

Comment:
Just a note, I think we can do without this package from Golang 1.13 onwards, see https://blog.golang.org/go1.13-errors. `fmt.Errorf("yaddda: %w", err)` with the `%w` directive does what `errors.Wrap(err, "yadda")` does, without losing the information about the wrapped error.

I suppose we could refactor this in one sweep throughout our code base, not blocking this PR.